ray-operator/config/samples/ray-service.mobilenet.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: ray.io/v1alpha1
+kind: RayService
+metadata:
+  name: rayservice-mobilenet
+spec:
+  serviceUnhealthySecondThreshold: 300 # Config for the health check threshold for service. Default value is 60.
+  deploymentUnhealthySecondThreshold: 300 # Config for the health check threshold for deployments. Default value is 60.
+  serveConfig:
+    importPath: mobilenet.mobilenet:app
+    runtimeEnv: |
+      working_dir: "https://github.com/ray-project/serve_config_examples/archive/master.zip"
```

Comment:
Could use specific commit hash instead of `master` here? That stops any regressions in the `serve_config_examples` repo from affecting this example.

Reply:
Good idea. Perhaps we can create a release for serve_config_examples with every KubeRay release.

Reply:
Updated https://github.com/ray-project/kuberay/pull/1175/commits/ced592acd4367cd68b4b5b3953d23016d0a9ab73. Point to this commit https://github.com/ray-project/serve_config_examples/commit/b393e77bbd6aba0881e3d94c05f968f05a387b96.